### DISCUSSION TITLE
Players Icon

### DISCUSSION THREAD

**Discussion by mirmen** [JLyne/LiveAtlas#461] [Q&A]
Hi. I used squaremap map (v 1.1.3) and 
squaremap-skins addon (https://github.com/jpenilla/squaremap-addons/tree/master/addons/skins) 

after installing LiveAtlas all player icons became Steve, I do not quite understand what I need to use as a placeholder because the {name} url does not work ( I used https://cravatar.eu/helmavatar/{name} )

sorry for my english

**Comment by JLyne** [comment]
You're not doing anything wrong, LiveAtlas doesn't support `{name}` in Squaremap's `heads-url`. I've now fixed this in #462.
